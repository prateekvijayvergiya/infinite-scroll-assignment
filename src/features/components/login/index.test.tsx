import Component from './'
import makeSnapshot from './../../testing/makeSnapshot'

jest.mock('@material-ui/core/Grid', () => 'Grid')
jest.mock('@material-ui/core/TextField', () => 'TextField')
jest.mock('@material-ui/core/Button', () => 'Button')
jest.mock('@material-ui/lab/Alert', () => 'Alert')

describe('Login', () => {
  it('renders correctly', () => {
    makeSnapshot(<Component />)
  })
})
