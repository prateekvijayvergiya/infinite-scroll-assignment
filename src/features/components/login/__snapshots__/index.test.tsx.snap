// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<Grid
  alignItems="center"
  container={true}
  direction="column"
  justify="center"
  spacing={4}
>
  <Grid
    item={true}
  >
    <TextField
      onChange={[Function]}
      placeholder="Enter username"
      value=""
    />
  </Grid>
  <Grid
    item={true}
  >
    <TextField
      onChange={[Function]}
      placeholder="Enter password"
      type="password"
      value=""
    />
  </Grid>
  <Grid
    item={true}
  >
    <Button
      color="primary"
      onClick={[Function]}
      variant="contained"
    >
      Login
    </Button>
  </Grid>
</Grid>
`;
